client/web-sveltekit/src/lib/fuzzyfinder/FuzzyFinderContainer.svelte

```diff
@@ -38,12 +49,18 @@
         ignoreInputFields: false,
         handler: event => {
             event.stopPropagation()
-            open = true
-            finder?.selectTab('files')
+            $fuzzyFinderState = true
+            $scopeState = 'files'
             return false
         },
     })
 
+    scopeState.subscribe(tab => {
+        if (tab) {
+            finder?.selectTab(tab)
+        }
+    })
```

Comment:
Likely not a problem since this component currently only gets destroyed when the page closes, but still: If you subscribe manually you should also unsubscribe:

```suggestion
    onDestroy(scopeState.subscribe(tab => {
        if (tab) {
            finder?.selectTab(tab)
        }
    }))
```

Or you could use a reactive statement:

```suggestion
    $: if (tab) {
        finder?.selectTab(tab)
    }
```